### ISSUE TITLE
Add support for FASTA format

### ISSUE THREAD

**Issue by drivenbyentropy** [drivenbyentropy/aptasuite#4]

**Comment by bioinformatics-deakin** [comment]
Thanks, this is working now in fastq format. I notice that the 'primer3' option in the config file als ahs to be the reverse complement of the reverse primer... which is not really the standard way to write primers. IT might be worth putting a note on that in the manual. Cheers.

**Comment by drivenbyentropy** [comment]
Good idea, I will add a note to the manual that the 3' primer is always to be provided in 5' to 3' orientation of the forward run. 

Thanks for pointing this out.
